src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -50,11 +78,12 @@ THE SOFTWARE.
                                     <j:set var="it" value="${item}"/>
                                     <j:set var="instance" value="${null}"/>
                                     <j:set var="help" value="${descriptor.helpFile}"/>
+                                    
                                     <j:if test="${help != null}">
-                                        <tr>
-                                            <td colspan="3"/>
+                                        <local:row>
+                                            <local:td />
```

Comment:
is this actually needed on a tables to divs core? what does the UI render like on new versions?

this is just adding an empty div

Reply:
From a quick test with 2.276, with and without this change, after tables-to-divs the help icon for the step itself and the first parameter of the step itself end up side by side. Before tables-to-divs, the help icon for the step itself was one line up (it has always looked kind of weird; there might be a Jira ticket about this). If you expand the help text, each icon gets its own line as before and things look normal.

<img width="1099" alt="Screen Shot 2021-01-27 at 16 59 45" src="https://user-images.githubusercontent.com/1068968/106059712-1871d500-60c1-11eb-9511-f2b33873c095.png">

If I change the whole section to the following (the local taglib would have to be tweaked to be able to add the `br` tag only after tables-to-divs), then the post-table-to-divs appearance looks pretty close to the previous appearance:

```
<div class="tr">
    <f:helpLink url="${help}"/>
</div>
<f:helpArea/>
<br/>
```

There might be a better way to accomplish the same result. Here's a screenshot with that change:

<img width="1088" alt="Screen Shot 2021-01-27 at 17 10 32" src="https://user-images.githubusercontent.com/1068968/106060858-9bdff600-60c2-11eb-87d5-2d2b464f30cc.png">

Reply:
I was able to fix it with:

```
                                    <j:if test="${help != null}">
                                        <div class="setting-name help-sibling snippetizer-help">
                                            <f:helpLink url="${help}"/>
                                        </div>
                                        <f:helpArea/>
                                    </j:if>
```

![image](https://user-images.githubusercontent.com/21194782/106111824-6ed42780-6144-11eb-98c7-6dea7401b1c7.png)

![image](https://user-images.githubusercontent.com/21194782/106111840-7398db80-6144-11eb-9a96-071235d4b387.png)

Don't have time to push a fix right now though